emiluz/README.md

```diff
@@ -0,0 +1,148 @@
+### Homework #1
+(Due Monday 7/20)
+
+You need to turn in this homework by cloning this repo, making a new branch with your solutions, pushing to github, and issuing a pull request.
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_30% of total time_)
+Predict the output of each of the following expressions, and explain the rules which determine that answer.  (_Hint: The rules for operator `==` are complicated, but do your best!_)
+
+If the output depends on the value or type of variable _x_, identify the conditions (what _x_ is) when the expression will output true (or false, if that's simpler).  Assume the cases are independent, and _x_ is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
+
+a) "1" == 1 true. One is equal to One.
+
+b) "1" === 1 false. String One is not strict equal to One.
+
+c) x == 'x' The X is not defined therefore X is not equal to string.
+
+d) x == (x+'') The X is not defined.
+
+e)'' == ' ' The answer is false, null
+
+f) x = true
+
+g) var x; x == 'undefined'` false. X is not equal to undefined
+
+h) '9'<'10' false. nine is not greater than 10
+
+i) typeof x + 1 === "number" false. undefined1 is not strict equal to number
+
+j)typeof x % 2 === "number" false. NaN is not strict equal to number
+
+k) typeof (x % 2) === "number" false. X is undefined therefore is not strict equal to number
+
```

Comment:
Even though x is undefined, the % operator always returns a number or NaN. Because of the ()'s, this operation occurs before typeof is called, so typeof(NaN) will return "number", which === "number", returning true! This will still work, even if x is a string or any other number.